front-end/src/services/socketService.js

```diff
@@ -1,5 +1,10 @@
 import io from 'socket.io-client';
 
-const URL = 'https://teacher-platform.onrender.com';
+let URL;
```

Comment:
Could be rewrite to `
let URL = 'https://teacher-platform.onrender.com';
if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
  URL = 'http://localhost:5000';
}`

Reply:
done